### DISCUSSION TITLE
Curl Error while Installing

### DISCUSSION THREAD

**Discussion by HarsshilR** [SpotX-Official/SpotX#318] [Errors]
I tried installiing all types of installations and even ran the commands on powershell but it always shows "Unable to connect your server , Please check your internet connection" & "Curl Error" while installing. Please help.
![image](https://user-images.githubusercontent.com/96742589/219679662-8803ef11-f8a6-48e5-8eab-b304c88bcfcd.png)

**Comment by amd64fox** [comment]
run this command in PowerShell terminal:
```ps1
curl.exe -s -w "%{http_code}" -o /dev/null https://download.scdn.co/upgrade/client/win32-x86/spotify_installer-1.2.5.1006.g22820f93-1078.exe --retry 2 --ssl-no-revoke
```

and send response here
